matrixbot/matrix.py

```diff
@@ -288,15 +288,20 @@ def send_private_message(self, user_id, message, room_id=None):
                              user_room_id, message)
 
     async def loop(self):
+        loop_pool = []
+        loop_max = 10
```

Comment:
Maybe loop_max could be a configurable param?

Reply:
I will left hardcoded for now but I will have this comment in consideration creating this issue: https://github.com/psaavedra/matrix-bot/issues/57

Reply:
:+1: